index.html

```diff
@@ -12,32 +12,20 @@
 <body>
   <div class="container">
     <div class="row">
-      <div class="col-lg-6 col-md-12">
-        <!-- ko foreach: toDoLists -->
-        <div class="row toDoList">
-          <!-- ko component: 'toDoList' -->
-          <!-- /ko -->
-        </div>
-        <button class="btn btn-block btn-danger" data-bind="click: $parent.removeToDoList">Remove</button>
+      <!-- ko foreach: widgets -->
+      <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">

```

Comment:
For each widget in the widget array, we output a column. The width of the column depends on the screen size, but is the same for all widget types.